Fix:

main.c:88:43: error: implicit declaration of function 'unlink' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- main.c.orig	2006-10-16 13:16:16.000000000 -0500
+++ main.c	2021-02-15 06:24:41.000000000 -0600
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #ifndef __WIN32__  /*rwj  --  make portable*/
 #include <signal.h>
